app/build.gradle

```diff
@@ -127,7 +106,9 @@ android {
         ccc36c3 {
             dimension defaultDimension
             applicationId "info.metadude.android.congress.schedule"
-            signingConfig signingConfigs.ccc36c3
+            if (hasSigningConfig('ccc36c3')) {
+                signingConfig signingConfigs.ccc36c3
+            }
```

Comment:
It would be convenient if the signing config name is automatically deduced from the name of the current product flavor. Something like this pseudo code ...

``` groovy
if (hasSigningConfig(this.name)) {
    signingConfig signingConfigs[this.name]
}
```

Then there is no cost of maintaining this.